src/tester.rs

```diff
@@ -349,6 +349,9 @@ pub(crate) struct TestCmd<'a> {
     pub status: Option<Status>,
     pub stderr: Option<Vec<&'a str>>,
     pub stdout: Option<Vec<&'a str>>,
+    /// A list of custom command line arguments which should be passed when
+    /// executing the test command.
+    pub args: Option<Vec<String>>,
```

Comment:
Since (under the hood) a `Vec` doesn't allocate anything until you push to it, it would probably be easier to make this `Vec<String>`. I think that will simplify some other code?

Reply:
Yeah that makes sense. See what you think now.